chef/cookbooks/neutron/recipes/common_agent.rb

```diff
@@ -214,10 +242,58 @@
         Chef::Log.info("#{name} usurped #{res[1].join(", ")} routes from #{bound_if}") unless res[1].empty?
       end
     end
+
+    execute "Neutron: add #{bound_if} to #{name}" do
+      command "ovs-vsctl del-port #{name} #{bound_if} ; ovs-vsctl add-port #{name} #{bound_if}"
```

Comment:
Me neither. AFAIK it's there from the beginning, but I guess it's a good opportunity to cleanup. :)